data/transport missions.txt

```diff
@@ -3469,3 +3469,105 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
+	name "Escort Hauler prototype to formal dinner"
+	description "Escort the Testament to <destination> to deliver sushi for a formal dinner."
+	source
+		planet "Deep"
+	destination "Skymoot"
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on offer
+		conversation
+			`A lady in a formal business suit walks gracefully over to you.`
+			`	"Captain <last>, Captain <last>! It is so nice to meet you here!"`
```

Comment:
Changed to single paragraph.

The original text did have an explanation on why the lady knew the captains name, but it was removed earlier in 43593b255d7ccbe6033317a3449d8703ca847daf as part of the review process. It also explains a little bit on how the player got her attention (which was more by accident).

Reply:
I think the "captain is famous because of combat rating" thing is getting a bit over-used in the game. 

@Amazinite Can you have a look at the explanation I had put in 43593b255d7ccbe6033317a3449d8703ca847daf and do a suggestion if that can be used here?

Reply:
I'd say that that is actually a fine explanation to have. It's basically just being honest in saying "Yeah, I'm just buttering you up."

Reply:
I re-added it (without the honor part which I think Pointedstick considered unfitting).